src/NUnitFramework/framework/Constraints/Tolerance.cs

```diff
@@ -32,6 +32,9 @@ namespace NUnit.Framework.Constraints
     /// type that supports taking a difference between two 
     /// objects and comparing that difference to a value.
     /// </summary>
+#if !NETSTANDARD1_3 && !NETSTANDARD1_6
+    //[Serializable]
```

Comment:
I can't imagine this is what you intended. 😸

Reply:
Hmmm, neither can I....

Reply:
Commit has been amended.